src/gui/torrentoptionsdialog.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Bittorrent Client using Qt and libtorrent.
- * Copyright (C) 2011  Christian Kandeler
- * Copyright (C) 2011  Christophe Dumez <chris@qbittorrent.org>
+ * Copyright (C) 2006  Christophe Dumez <chris@qbittorrent.org>
```

Comment:
Is this a valid copyright now?

Reply:
Sure i don't care.

Reply:
wtf? Why was the **deletion** of other people's copyright years allowed? Was the file rewritten so much that their code isn't here anymore?

Reply:
Seems git incorrectly show this file as rewritten `updownratiodialog.h`. Looking at its diff and diffs of the accompanying .`cpp`/`.ui` files, it is clear that this is something created from scratch (am I wrong?).
The question is why the author did not want to indicate himself in the copyright, but indicated Christophe Dumez. We did not focus on this...
To be honest, I do not understand the intricacies of copyright, and we do not have any declared policy about it to be guided by it.